pkg/pillar/types/global.go

```diff
@@ -165,6 +178,10 @@ const (
 	UsbAccess GlobalSettingKey = "debug.enable.usb"
 	// AllowAppVnc global setting key
 	AllowAppVnc GlobalSettingKey = "app.allow.vnc"
+	// EveMemoryLimitInBytes global setting key
+	EveMemoryLimitInBytes GlobalSettingKey = "memory.eve.limit.bytes"
+	// IgnoreMemoryCheckForApps global setting key
+	IgnoreMemoryCheckForApps GlobalSettingKey = "memory.apps.ignore.check"
```

Comment:
I don't see these being documented in CONFIG-PROPERTIES.md. Please add

Reply:
Added